cmake/importSDL2.cmake

```diff
@@ -50,11 +56,30 @@ function(build_SDL2)
     set(SDL2_FOUND "From build_SDL2" PARENT_SCOPE)
 endfunction()
 
+function(link_to_system_SDL2)
```

Comment:
Why do we need to create a symlink? Pharo should look in the system paths, that's all.

Reply:
If not symlinked the Pharo UI won't start because #findSDL2 cannot find the library.

Do you mean to change findSDL2 to scan all possible system paths for all platforms?

Reply:
I found a way, updating knownPaths: 

```smalltalk
knownPaths

	^ #(
	'/usr/lib'
	'/usr/local/lib'
	'/opt/homebrew'
	'/opt/homebrew/lib'
	)
```
(However I don't know if values for Unix32, Unix64 should also be updated).

It seems to me that this way we are replicating logic already implemented by CMake modules. With CMake we could ship a symlink to a standard location, however, replicating the logic with the FFI library finder it opens the door to unexpected issues combined with the search order (such as the one migrating from platforms which doesn't automatically clean old libraries).

Reply:
No, shipping symlinks will not work.
First, Depending on how files are untar'd or unzipped, symlinks can be resolved and files copied, which would create a mess.

Also, symlinks will only work if you are building your own VM.
Otherwise, we may build a VM with a symlink to `/opt/homebrew` but the local installation of somebody else is in `/usr/lib/` because he/she compiled it his/herself, thus it will not work.